cpu/stm32_common/periph/flashpage.c

```diff
@@ -185,14 +186,16 @@ void flashpage_write_raw(void *target_addr, const void *data, size_t len)
     }
 
     /* clear program bit again */
+#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F1) || \
+    defined(CPU_FAM_STM32L4)
     CNTRL_REG &= ~(FLASH_CR_PG);
     DEBUG("[flashpage_raw] write: done writing data\n");
+#endif
```

Comment:
The DEBUG message above is now only displayed with F0/F1/L4 but not for L0/L1. It should be put after.
I would also not put this change in 5e0c86720013de864829e3e0edc10ace3f058e0d because it seems unrelated.

Reply:
You are right about the `DEBUG` message. But it is related since I'm removing the definition of `FLASH_CR_PG` Therefore I need to excluded it from compilation and that is the reason for the `#ifdefined`. Should I split guarding and removing `FLASH_CR_PG` into two commits?

Reply:
I see, let's keep the changes in the same commit then. You still need have to fix the commit message: it's full of typos :)

Reply:
The DEBUG line must still move out of the define (preferably after).

Reply:
damn, I was concentrating on the typos sorry

Reply:
Ok I think now it should be ok .. sorry for missing this.